web3/manager.py

```diff
@@ -102,6 +98,26 @@ def _make_request(self, method, params):
                 )
             )
 
+    async def _make_async_request(self, method, params):
+        """TODO: Can this be made dry? Going for dumb solution for now.
+        """
+        for provider in self.providers:
```

Comment:
We *should* be able to drop this loop in `v5` by removing the multi-provider API.  cc @kclowes